source/_components/sensor.fritzbox_callmonitor.markdown

```diff
@@ -27,6 +27,13 @@ $ sudo apt-get install libxml2-dev libxslt-dev \
   python3-setuptools zlib1g-dev build-essential
 ```
 
+If you installed Home Assistant in a virtualenv, also run the following command inside it. 
+Be patient this will take a while.
+```bash
+pip3 install lxml
```

Comment:
Why not simply add this to the instructions above?

Reply:
A user might be tempted to run all the commands in a sequence. But this is not possible, because he would have to bin/activate the VirtualEnv in between first. Also, this additional hint apparently is valid only for VirtualEnv installations. I cannot explain why. But it's stated in the links I'll post in my general answer below. 

I suggest to keep it separate unless we (i.e. not me) know better.